tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -73,6 +75,7 @@
     private final Map<String, Object> properties = new TreeMap<>();
     private long seed;
     private int maxNode;
+    private String vehicle;
```

Comment:
Here I thought the measurements are running only for one vehicle anyway (and if we want to test others we might as well start the measurement again for the other vehicle), so it makes sense to keep the vehicle as a class variable (just like weighting) rather than pass it with every querysetting?

Reply:
In the CustomWeighting branch I already use `profile` and then I do not want the vehicle in the request:  https://github.com/graphhopper/graphhopper/pull/1841/files#diff-cd749cc545f8b84137c2b75b2d174b72 but probably this can be still done somehow.

Reply:
Ok yes vehicle and weihgting should be mostly removed from measurement and only profile should be left then?

Reply:
Yes, but let's do this later :)

Reply:
There isn't really anything to do. vehicle&weighting are only used to create the profile initially.